.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I hoped we could avoid dependency on intellij settings, but i guess it's better this way